plone/app/vocabularies/types.py

```diff
@@ -298,9 +299,13 @@ def __call__(self, context):
             for t in ttool.listContentTypes()
             if t not in BAD_TYPES
         ]
-        items.sort()
-        items = [SimpleTerm(i[1], i[1], i[0]) for i in items]
-        return PermissiveVocabulary(items)
+        
+        print(items)
```

Comment:
Remove this `print`

Reply:
seems like unidecode is not added to setup.py requirements

```
Missing requirements
====================
     unidecode.unidecode
```

Reply:
the reason for other two tests being failed is unknown to me... Any help to tackle the failing test is greatly appreciated... @wesleybl

Reply:
> the reason for other two tests being failed is unknown to me... Any help to tackle the failing test is greatly appreciated

See: https://github.com/plone/plone.app.vocabularies/pull/93#discussion_r1688272276